api/basic/helper/response.ts

```diff
@@ -74,3 +74,13 @@ export function toGetPinResponse (
     }
   }
 }
+
+/**
+ * Convert psa pin status to cluster pin status!
+ */
+function toClusterTrackerStatus (pinStatus?: Pin['status']): TrackerStatus {
```

Comment:
Confirming we don't need any more mapping from the ones in https://github.com/nftstorage/ipfs-cluster/blob/74dfd779329045d093267cf3a819ff9a8d73af2f/src/interface.ts#L198-L251, with Pinning API spec https://ipfs.github.io/pinning-services-api-spec/#operation/getPins `Items Enum: "queued" "pinning" "pinned" "failed"`. Therefore looks good 👍🏼